feedlist/feedlist.go

```diff
@@ -62,20 +62,27 @@ func fetchFeedAndParse(url string) (*gofeed.Feed, error) {
 	return feed, nil
 }
 
+const (
+	fetchMaxTries   = 5
+	fetchRetryDelay = 200 * time.Millisecond
+)
+
 // Feed takes an URL as input, and returns a *gofeed.Feed.
 func Feed(url string) (*gofeed.Feed, error) {
 	var feed *gofeed.Feed
 	var err error
 
-	// Try up to 5 times
-	for i := 0; i < 5; i++ {
+	// Try up to fetchMaxTries times
+	for i := 0; i < fetchMaxTries; i++ {
+		if i != 0 {
```

Comment:
I'd probably prefer the sleep to be unconditional, after the fetch.

No need to special-case `i!=0` then.

(Also I guess multiply the delay by the retry-count.  It's not exponential backoff, but it feels nicer than just the same (brief) delay each time.)